packages/asc-ui/src/components/StepByStepNav/StepByStepNavStyle.ts

```diff
@@ -53,6 +54,17 @@ export const OrderdedList = styled.ol<{
   }
 `
 
+export const ComponentWrapper = styled.div<{
+  breakpoint?: StepByStepNavProps['breakpoint']
+}>`
+  @media screen and ${({ breakpoint }) => breakpoint} {
+    display: inline-block;
+    height: ${activeItemSize}px;
+    visibility: hidden;
+    width: ${activeItemSize}px;
```

Comment:
Shouldn't `width` be 0? The current width causes steps with components to be 92px wide on screens < 1023px, whereas steps without components are 56px on smaller screens

Reply:
I've changed the width and height size to 0